src/js/jsx/help/FirstLaunch.jsx

```diff
@@ -62,6 +63,18 @@ define(function (require, exports, module) {
             }
         },
 
+        /**
+         * Calls for the URL to be opened and then logs the headlights data
+         *
+         * @param {string} url
+         * @param {string} name
+         *
```

Comment:
(To see an example, check out @shaoshing's open Libraries analytics PR. I stole the idea from him.)